phys/module_sf_noahdrv.F

```diff
@@ -330,19 +338,19 @@ SUBROUTINE lsm(DZ8W,QV3D,P8W3D,T3D,TSK,                      &
                                                         ALBEDO, &
                                                            ZNT
    REAL,    DIMENSION( ims:ime, jms:jme )                     , &
-            INTENT(OUT)      ::                         NOAHRES
+            INTENT(INOUT)    ::                         NOAHRES
    INTEGER, INTENT(IN)       ::                         OPT_THCND
 
 ! Noah UA changes
    LOGICAL,                                INTENT(IN)  :: UA_PHYS
-   REAL,    DIMENSION( ims:ime, jms:jme ), INTENT(OUT) :: FLX4_2D,FVB_2D,FBUR_2D,FGSN_2D
+   REAL,    DIMENSION( ims:ime, jms:jme ), INTENT(INOUT) :: FLX4_2D,FVB_2D,FBUR_2D,FGSN_2D
    REAL                                                :: FLX4,FVB,FBUR,FGSN
 
    REAL,    DIMENSION( ims:ime, jms:jme )                     , &
-               INTENT(OUT)    ::                        CHKLOWQ
+               INTENT(INOUT)  ::                        CHKLOWQ
```

Comment:
Both variables NOAHRES and CHKLOWQ appear only on the LHS of equations, should they be declared (INOUT)? Did it cause some problem on the MPAS side?